sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -2259,6 +2264,26 @@ public ReadChangeStream withCreateOrUpdateMetadataTable(boolean shouldCreate) {
       return toBuilder().setCreateOrUpdateMetadataTable(shouldCreate).build();
     }
 
+    /**
+     * Returns a new {@link BigtableIO.ReadChangeStream} that removes the replication delay
```

Comment:
Suggestion:

"Returns a new {@link BigtableIO.ReadChangeStream} that overrides the replication delay adjustment duration with the provided duration."

Reply:
Thanks, this is better. Updated